### ISSUE TITLE
installation instructions to "automake --add-missing" fails

### ISSUE THREAD

**Issue by rmaldersoniii** [Homebrew/homebrew#43386]
Attempting to install Homebrew on MacOS X 10.8.5 following instructions relating to Hercules emulator:

> Install Xcode from the App Store.
> Install Homebrew using the procedure described at http://brew.sh/
> Use these commands to install pre-requisite software: 
> brew install autoconf 
> brew install automake 
> brew install gnu-sed 
> automake --add-missing
> ^^^^^^^^^^^^^^^^^^^
> Proceed with Building from source - Linux and Mac OS X below.

The step "underlined" with "^" fails with the message *automake: error: 'configure.ac' is required* .

I don't see anything documenting the "--add-missing" switch to automake, so I have no quick way to check whether this cookbook method is broken.

**Comment by dunn** [comment]
> Install Xcode from the App Store.

Let's start here. Do you have Xcode installed, or the Command-Line Tools from https://developer.apple.com/downloads/ ?

**Comment by dunn** [comment]
Where did these instructions come from?  Are you trying to install Homebrew, or install software with Homebrew?

**Comment by DomT4** [comment]
We do actually have a [hercules](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/hercules.rb) formula if you just prefer to use that.

It's perhaps unclear upstream, you need to run `automake --add-missing` inside the source directory of what you're trying to compile. It's a unique process for each build script.

```
cd hyperion-master
automake --add-missing
```

etc.

**Comment by rmaldersoniii** [comment]
dunn: The instructions came from the Hercules page, for building the Hercules emulator from source. I had already installed Xcode, so I went to the Homebrew URL listed, and pasted the installation invocation into a Terminal.app window (using Ruby to call curl). I then followed the Homebrew instructions to run "brew help" and "brew doctor", corrected some things left over from upgrading MacOS from Snow Leopard to Mountain Lion, and put the Homebrew executables into my PATH. Then I successfully ran the 3 installs listed above, followed by the automake invocation.

DomT4: The Hercules folks are the source of the confusion here. They make no reference to their own source directory until the next set of instructions (the section which is common to Linux and MacOS builds). I'll try running the automake in that directory, and go after them if I encounter further issues.

Thank you all (including those who contacted me privately to ask if I had installed Xcode and the command line tools--a fair question, I suppose). I apologize for the inconvenience.
